src/renderer/components/archive/entry-icon.js

```diff
@@ -0,0 +1,20 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import styles from '../../styles/entry-icon';
+import defaultIcon from '../../styles/img/lock-black.svg';
+
+const EntryIcon = props => {
```

Comment:
Instead of doing this, we can perhaps do styled-components:

```js
const EntryIcon = styled.div`
  background-image: `url(${props => props.icon ? props.icon : defaultIcon})`;
`
```
etc. Since we are trying to use styled-components in the project instead of css